registration/templates/registration/register.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+{% block title %}
+    {{ title }}
```

Comment:
This guy is abstract enough that we can use it for pretty much every type of registration.